test/Tools/docGeneratorTestFiles/sampledictionarycontent/Dynamo_Nodes_Documentation.json

```diff
@@ -2992,6 +2992,12 @@
     "dynFile": ["Output"],
     "folderPath": "Input/Basic/Action",
     "inDepth": "Output will create an output port for a custom node. The syntax for an output node is simply its name, along with an optional custom comment. In the example below, an output named Percentage is created with a custom comment. It is important to note that this node is only available while creating a custom node."
+}, {
+    "Name": "Custom Dropdown",
+    "imageFile": ["Custom Dropdown"],
+    "dynFile": ["Custom Dropdown"],
+    "folderPath": "Input/Basic/Action",
+    "inDepth": "This node allows you to pre-define a dropdown menu with with an arbitrary number of custom items. Each item consists of a label and a value. The label is always a string, while the value can be integer, number of string"
```

Comment:
type (`or` string?) - Does this allow return types to be arbitrary objects or only int,double,string?